device/ingrasys/x86_64-ingrasys_s8900_54xc-r0/INGRASYS-S8900-54XC/port_config.ini

```diff
@@ -0,0 +1,55 @@
+# name          lanes             alias
+Ethernet0       1                 Ethernet0 
+Ethernet1       2                 Ethernet1 
+Ethernet2       3                 Ethernet2 
+Ethernet3       4                 Ethernet3 
+Ethernet4       5                 Ethernet4 
+Ethernet5       6                 Ethernet5 
+Ethernet6       7                 Ethernet6 
+Ethernet7       8                 Ethernet7 
+Ethernet8       9                 Ethernet8 
+Ethernet9       10                Ethernet9 
+Ethernet10      11                Ethernet10
+Ethernet11      12                Ethernet11
+Ethernet12      21                Ethernet12
+Ethernet13      22                Ethernet13
+Ethernet14      23                Ethernet14
+Ethernet15      24                Ethernet15
+Ethernet16      33                Ethernet16
+Ethernet17      34                Ethernet17
+Ethernet18      35                Ethernet18
+Ethernet19      36                Ethernet19
+Ethernet20      37                Ethernet20
+Ethernet21      38                Ethernet21
+Ethernet22      39                Ethernet22
+Ethernet23      40                Ethernet23
+Ethernet24      41                Ethernet24
+Ethernet25      42                Ethernet25
+Ethernet26      43                Ethernet26
+Ethernet27      44                Ethernet27
+Ethernet28      49                Ethernet28
+Ethernet29      50                Ethernet29
+Ethernet30      51                Ethernet30
+Ethernet31      52                Ethernet31
+Ethernet32      53                Ethernet32
+Ethernet33      54                Ethernet33
+Ethernet34      55                Ethernet34
+Ethernet35      56                Ethernet35
+Ethernet36      65                Ethernet36
+Ethernet37      66                Ethernet37
+Ethernet38      67                Ethernet38
+Ethernet39      68                Ethernet39
+Ethernet40      69                Ethernet40
+Ethernet41      70                Ethernet41
+Ethernet42      71                Ethernet42
+Ethernet43      72                Ethernet43
+Ethernet44      81                Ethernet44
+Ethernet45      82                Ethernet45
+Ethernet46      83                Ethernet46
+Ethernet47      84                Ethernet47
+Ethernet48      85,86,87,88       Ethernet48
+Ethernet52      97,98,99,100      Ethernet52
+Ethernet56      101,102,103,104   Ethernet56
+Ethernet60      105,106,107,108   Ethernet60
+Ethernet64      109,110,111,112   Ethernet64
+Ethernet68      117,118,119,120   Ethernet68
```

Comment:
The `alias` column here is optional, and seeing as your aliases are exactly the same as their corresponding SONiC port names, the aliases serve no purpose for this device. You can safely remove the `alias` column in this file.